packages/nimble-components/src/table/specs/table-column-width-hld.md

```diff
@@ -0,0 +1,157 @@
+# Table Column Widths
+
+## Problem Statement
+
+We need to provide users the means for changing the widths of individual columns as needed. This should be able to be accomplished both through interactive means as well as via a programmatic API.
+
+## Links To Relevant Work Items and Reference Material
+
+- [#873 Programmatically resize column width](https://github.com/ni/nimble/issues/873)
+- [#846 Interactively resize column width](https://github.com/ni/nimble/issues/846) 
```

Comment:
I think this is missing some discussion of interactive considerations:
1. mouse cursors
2. accessibility. I guess column widths only matter to sighted users so we don't need to reflect column widths in the accessibility tree? What about sighted users who don't use a mouse; do we need to provide a way to resize columns with keyboard/other assistive technology? Whatever we decide here, we should document it, even if the answer is "we don't plan to do anything"
3. maybe also things like live resizing, hover states, grab handle sizes, etc, though those could probably just live in the VxD spec

Reply:
I've mentioned the "accessible" interactive sizing as being out-of-scope for this HLD, as whatever that might involve should not be in conflict with the API being proposed, nor the mouse-based interactive sizing that is described.

I've also added an 'Interactive visual states' section under **Implementation considerations** to point them to the design document for the various visual considerations you mention.